.nextmv/release/main.py

```diff
@@ -241,5 +251,21 @@ def bump_marketplace_version(
     return f"v{major}.{minor}.{patch}"
 
 
+def notify_slack(url: str, app: str, version: str, marketplace_version: str):
+    """
+    Notify Slack channel about release.
+    """
+
+    response = requests.post(
+        url,
+        json={
+            "text": f"Release notification - community-apps/{app} {version} / marketplace {marketplace_version}",
```

Comment:
I think we should either only notify for prod or log the environment that is being released in here.

Reply:
Yeah, totally. I somehow missed the distinction here. The parameter is optional, so, I will simply drop it in the workflow if not targeting prod. :relaxed:

Reply:
Thanks for noticing! :hugs: